website/snippets/_cloud-environments-info.md

```diff
@@ -42,6 +42,12 @@ For improved reliability and performance on your job runs, you can enable dbt Cl
 
 dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 
 
+Below lists the situations when dbt Cloud uses the cached copy:
+
+- Git authentication fails.
+- There are syntax errors in the `packages.yml` file. To catch these errors sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).  
+- A package is incompatible with the dbt version being used. To catch this incompatibility sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).  
+
```

Comment:
For completeness, don't we also want to include failures of 3rd party services (github, dbt package hub, etc.) in this section?

Reply:
great flag! i agree with you. will update shortly